ui/shared/TransactionSigner.qml

```diff
@@ -0,0 +1,74 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import "../imports"
+
+Item {
+    id: root
+    height: signingPhraseItem.height + signingPhrase.height + txtPassword.height + Style.current.smallPadding + Style.current.bigPadding
+
+    property string signingPhrase: "not a real one"
+    property alias passwordInput: txtPassword
+    property string validationError: ""
+  
+    Item {
+        id: signingPhraseItem
+        anchors.horizontalCenter: parent.horizontalCenter
+        height: labelSigningPhrase.height 
+        width: labelSigningPhrase.width + infoButton.width + infoButton.anchors.leftMargin
+
+        StyledText {
+            id: labelSigningPhrase
+            color: Style.current.secondaryText
+            font.pixelSize: 15
+            text: qsTr("Signing phrase")
+        }
+
+        IconButton {
+            id: infoButton
+            clickable: false
+            anchors.left: labelSigningPhrase.right
+            anchors.leftMargin: 7
+            anchors.verticalCenter: parent.verticalCenter
+            width: 13
+            height: 13
+            iconName: "info"
+            color: Style.current.lightBlue
+        }
```

Comment:
This icon button doesn't do anything yet. I wanted to add a `ToolTip` but those only work on control elements. Unfortunately, our `IconButton` isn't a control element but a `Rectangle` so we need to refactor `IconButton` first to be a proper `Button` (as it should be)